api/ticket/shared_drafts.rst

```diff
@@ -0,0 +1,1838 @@
+Shared drafts
+*************
+
+Show
```

Comment:
There is one additional currently undocumented route, `import_attachments`. Maybe you can check with @mantas if we should document this as well, and how it works.

Reply:
I got in touch with him and we considered it as not important. He states that there is probably just one use case which is not very feasible (user opens ticket and draft > copies the form ID > calls the API with this ID to import an attachment).

But thanks for your review and keeping your eyes open anyway :-)